policyengine_us/parameters/gov/states/nj/tax/exemptions/regular/amount.yaml

```diff
@@ -0,0 +1,19 @@
+description: NJ provides a regular exemption of $1k for individuals and $2k for couples.
+metadata:
+  unit: currency-USD
+  label: NJ Regular Exemption
+  reference:
+    - title: Instructions for Form NJ-1040
```

Comment:
Could you add a legal code reference?

Reply:
@ZiqianCui I think either of these should be good!

https://law.justia.com/codes/new-jersey/2022/title-54a/section-54a-3-1/

https://casetext.com/statute/new-jersey-statutes/title-54a-new-jersey-gross-income-tax-act/chapter-54a3-personal-exemptions-and-deductions/section-54a3-1-personal-exemptions-and-deductions

Reply:
I pasted the second source!